<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddonsEditor</class>
 <widget class="QWidget" name="AddonsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QScrollArea" name="addons_view">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>445</width>
        <height>328</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListView" name="list_view"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="add_addon">
       <property name="text">
        <string>Add addon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_addon">
       <property name="text">
        <string>Remove selected addons</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>add_addon</sender>
   <signal>clicked()</signal>
   <receiver>AddonsEditor</receiver>
   <slot>add_addon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>556</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remove_addon</sender>
   <signal>clicked()</signal>
   <receiver>AddonsEditor</receiver>
   <slot>remove_selected_addons()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>556</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
